microsoft-edge/extensions-chromium/getting-started/manifest-format.md

```diff
@@ -87,12 +87,13 @@ For reference information about each field, see [Manifest file format (V3)](http
     "chrome_style": true,
     "page": "options.html"
   },
-  "permissions": ["tabs"],
+  "permissions": ["..."],
   "platforms": ...,
   "replacement_web_app": ...,
   "requirements": {...},
   "sandbox": [...],
   "short_name": "Short Name",
+  "side_panel": {...},
```

Comment:
@Bhuvana-Priya - check the JSON I added in MV3

* "action" - I kept it same as https://developer.chrome.com/docs/extensions/mv3/manifest/#overview
* "permissions"- I made it same as https://developer.chrome.com/docs/extensions/mv3/manifest/#overview
* Added new "side_panel" like https://developer.chrome.com/docs/extensions/mv3/manifest/#overview
* Didn't add "side_panel" to MV2 listing below it, b/c Chrome's archived MV2 "Manifest file format" page doesn't list "side_panel": https://developer.chrome.com/docs/extensions/mv2/manifest/

Reply:
closing b/c the 3 lines match Chrome page